rasa/core/policies/ted_policy.py

```diff
@@ -332,12 +332,13 @@ def get_default_config() -> Dict[Text, Any]:
             # ingredients in a recipe, but it doesn't make sense for the parts of
             # an address
             SPLIT_ENTITIES_BY_COMMA: SPLIT_ENTITIES_BY_COMMA_DEFAULT_VALUE,
+            # Max history of the policy, unbounded by default
+            POLICY_MAX_HISTORY: DEFAULT_MAX_HISTORY,
```

Comment:
🙌🏻 Could you please also add the policy priority here? I forgot to do so 🙌🏻 🙈

Reply:
will do